app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1114,32 +1114,42 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
-    var historyMetadataUIFeature by featureFlagPreference(
+    var historyMetadataUIFeature by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_history_metadata_feature),
-        default = FeatureFlags.historyMetadataUIFeature,
+        default = { appContext.components.analytics.features.homeScreen.isRecentExplorationsActive() },
         featureFlag = FeatureFlags.historyMetadataUIFeature || isHistoryMetadataEnabled
     )
 
     /**
      * Indicates if the recent tabs functionality should be visible.
      * Returns true if the [FeatureFlags.showRecentTabsFeature] and [R.string.pref_key_recent_tabs] are true.
      */
-    var showRecentTabsFeature by featureFlagPreference(
+    var showRecentTabsFeature by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_recent_tabs),
-        default = FeatureFlags.showRecentTabsFeature,
-        featureFlag = FeatureFlags.showRecentTabsFeature
+        featureFlag = FeatureFlags.showRecentTabsFeature,
```

Comment:
@jhugman  I wonder can remove the `FeatureFlags` and completely rely on the variables from nimbus?

Reply:
I think I'm okay with this as long as the lookup for `features.homeScreen` is done lazily.

Having said that, I think this is a product decision. For this issue, the brief was to have Nimbus toggling the default visibility of the new sections, not their availability.

Reply:
Perfect.
Just to confirm, is nimbus going to provide default(true) values for nightly as well ? As we want these sections still show in nightly while we experiment with toggling the on Beta.